scripts/mongodb/queries/student-language-stats.js

```diff
@@ -0,0 +1,27 @@
+load('bower_components/lodash/dist/lodash.js');
```

Comment:
Hmm, it's kind of a common thing in mongodb scripts. `load` runs a given script, in this case lodash. It's used in 10 other scripts, as well, and counting. Given that, where would be a good place to document that's what it's for?